### PR TITLE
Send response.data to bulk action event listeners

### PR DESCRIPTION
**Pull request by simonhamp**
Closes https://github.com/statamic/ideas/issues/549

### CODE REVIEW

File: resources/js/components/data-list/Actions.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default {
                     });
                 }
 
-                this.$emit('completed');
+                this.$emit('completed', response.data);
```
Comment (jasonvarga) [inline resources/js/components/data-list/Actions.js:59]: The first argument is the success boolean (see the one happening in the failure) so it should be `true`, then the response data.

**Review by jasonvarga — CHANGES_REQUESTED** [review]
How are you using these events? In a custom component?

**Comment by simonhamp** [comment]
@jasonvarga yeh, in [Gitamic](https://statamic.com/addons/simonhamp/gitamic) [here](https://github.com/simonhamp/Gitamic/blob/main/resources/js/components/Staged.vue#L94-L99). I'd love to be able to make the message more detailed based on information from the response.

File: resources/js/components/data-list/Actions.js (hunk 1 above)
Comment (simonhamp) [reply resources/js/components/data-list/Actions.js:59]: Ah ok, I guess that was implied as it's not explicitly passed here. Adding it explicitly...

**Comment by jasonvarga** [comment]
Hey Simon

Since you're not actually using our "actions" feature, this change isn't super beneficial since the response would always just be an empty array.

https://github.com/statamic/cms/blob/af5b35b5ac9294ce9879b01b6e96e5dbe05085df/src/Http/Controllers/CP/ActionController.php#L44

I'm going to close this, but check out the PR that I'm going to open shortly that'll give you the functionality you want.

**Comment by simonhamp** [comment]
Hmmm... ok I'll take another look... it's been a while but I was pretty sure my code runs from that code - I'm definitely using actions.

**Pull request closed without merging**